CLA signed by HvC

henkvancann opened the pull request:
Hopefully I won't eat more of your time with this first PR of mine.

henkvancann committed:
master version of the Blockchain Commons CLA signed by Henk van Cann

henkvancann committed:
standard CLA filled out and signed by Henk van Cann

ChristopherA reviewed (approved):
+1 Thank you for becoming a contributor!!

henkvancann committed:
standard CLA filled out and signed by Henk van Cann, second try with added email address

henkvancann commented:
I added the github global e-mail address to the GPG key and I added user.email address as being hvancann@2value.nl

Could you try again the verify the new signature in the same pull request?

Best, Henk

> On 7 Jul 2020, at 03:42, Christopher Allen <notifications@github.com> wrote:
> 
> 
> @ChristopherA approved this pull request.
> 
> +1 Thank you for becoming a contributor!!
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/BlockchainCommons/FullyNoded-2/pull/82#pullrequestreview-443506816>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AADVA7HA2K3UBEEUWPBMAT3R2J4PTANCNFSM4OSKIC5Q>.
>

henkvancann commented:
I added the github global git user.email address to the GPG key using GPG Keychain for Mac and I also added user.email address hvancann@2value.nl locally to the FN2 repo. I signed the commit again with the renewed key and pushed the commit in the same PR.

Could you pls try again to verify the new signature?
